components/layout/fragment.rs

```diff
@@ -1974,8 +1975,8 @@ impl Fragment {
                                         relative_containing_block_size: &LogicalSize<Au>,
                                         coordinate_system: CoordinateSystem)
                                         -> Rect<Au> {
-        // FIXME(pcwalton, #2795): Get the real container size.
-        let container_size = Size2D::zero();
+        // FIXME (mbrubeck): Get the real container size, taking vertical writing modes into account.
+        let container_size = Size2D(relative_containing_block_size.inline, relative_containing_block_size.block);
```

Comment:
This code doesn't know the relative containing block's writing mode, so it can't call to_physical.  I was going to fix this in a followup, either by adding parameters to thread the writing mode into this function along with the size, or by giving each flow/fragment a direct link to its containing block.